src/libs/TransactionUtils.js

```diff
@@ -141,6 +141,11 @@ function getUpdatedTransaction(transaction, transactionChanges, isFromExpenseRep
         shouldStopSmartscan = true;
     }
 
+    if (_.has(transactionChanges, 'category')) {
+        updatedTransaction.modifiedCategory = transactionChanges.category;
+        shouldStopSmartscan = false;
```

Comment:
We only ever edit one field at a time so this is not needed here as false is default
```suggestion
```